examples/download_multiple_assets.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 Planet Labs PBC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+"""Example of downloading multiple assets in parallel
+
+This is an example of getting two assets, activating them, waiting for them to
+become active, downloading them, then validating the checksums of downloaded
+files.
+
+[Planet Explorer](https://www.planet.com/explorer/) was used to define
```

Comment:
Is this true for this script and where are AOIs used here?

Reply:
Certainly was for the wildfire AOI, unsure about the river AOI, I took that from the guide, but I assume so.